.github/workflows/ci.yml

```diff
@@ -10,13 +10,19 @@ on:
       - master
 jobs:
   ci:
-    name: Run checks and tests over ${{matrix.otp_vsn}} and ${{matrix.os}}
+    name: Checks and tests on OTP-${{matrix.otp_vsn}}, ${{matrix.os}}, ${{matrix.profile}}
     runs-on: ${{matrix.os}}
 
     strategy:
       matrix:
-        otp_vsn: [20.3, 21.3, 22.3, 23.1]
+        otp_vsn: [20.3, 21.3, 22.3, 23.3, 24.0]
```

Comment:
I think we can drop OTP 20, as rebar3 is only supporting the last three OTP releases.

For this reason we have also removed OTP 20 from the testing matrix in Zotonic.

Reply:
`rebar3` version I'm bundling in this PR `3.15.2` works fine for all versions from 20 to 24. But maybe I'm just lucky. I was jsut thinking that #267 would have to drop OTP-20 (because `logger` was introduced in OTP-21), so, it can be done there?